DesktopModules/Vanjaro/UXManager/Extensions/Menu/YouTube/Controllers/SettingController.cs

```diff
@@ -19,7 +19,8 @@ internal static List<IUIData> GetData(int portalId, UserInfo userInfo)
         {
             Dictionary<string, IUIData> Settings = new Dictionary<string, IUIData>();
             string Site_ApiKey = PortalController.GetEncryptedString("Vanjaro.Integration.YouTube", portalId, Config.GetDecryptionkey());
-            string Host_ApiKey = HostController.Instance.GetEncryptedString("Vanjaro.Integration.YouTube", Config.GetDecryptionkey());
+            HostController hostController = new HostController();
+            string Host_ApiKey = hostController.GetEncryptedString("Vanjaro.Integration.YouTube", Config.GetDecryptionkey());
```

Comment:
Please refactor all code that is using hostController; instead use the new SettingManager.GetHostSetting

Reply:
changes done